src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  const splitString = sourceString.trim().split(';');
+
+  splitString.forEach((item) => {
+    const splitItem = item.trim().split(':');
+
+    if (splitItem.length === 2) {
+      const key = splitItem[0].trim();
+      const value = splitItem[1].trim();
+
+      result[key] = value;
+    }
+  });
+
+  return result;
```

Comment:
u can use `reduce` method for it. Try it

```suggestion
   return sourceString.split(';').reduce((style, declaration) => {
    const [key, value] = declaration.trim().split(':');

   ..............
   
    return style;
  }, {});
```